src/view-port-cache.js

```diff
@@ -0,0 +1,26 @@
+import {transient} from 'aurelia-dependency-injection';
+
+@transient()
+export class ViewPortCache {
+  controllers = {};
```

Comment:
@stoffeastrom - I can make that change. The issue solved by `Object.create(null)` will not be an issue here due to the format of the keys but it's good to be consistent.